lib/srv/discovery/database_watcher.go

```diff
@@ -75,15 +75,7 @@ func (s *Server) startDatabaseWatchers() error {
 			Origin:         types.OriginCloud,
 			Clock:          s.clock,
 			PreFetchHookFn: func() {
```

Comment:
Here and in the other hook fns that only call one function with no arguments and no return value:
```suggestion
			PreFetchHookFn: s.databaseWatcherIterationStarted,
```